src/charm_types.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
I think this shebang line could be removed

Reply:
I will remove that in the next pull request, thanks!